virtual-host-gatherer/virtual-host-gatherer.spec

```diff
@@ -18,16 +18,20 @@
 
 %global with_susecloud 0
 %define skip_python2 1
+%global __python /usr/bin/python3
+
+%{?!python_module:%define python_module() python3-%{**}}
+%{?!python_build:%define python_build %{expand:%py3_build}}
+%{?!python_install:%define python_install %{expand:%py3_install}}
 
-%{?!python_module:%define python_module() python-%{**} python3-%{**}}
 Name:           virtual-host-gatherer
 Version:        1.0.23
-Release:        1
+Release:        0
 Summary:        Gather virtualization information
 License:        Apache-2.0
 Group:          Development/Languages/Python
 URL:            https://github.com/uyuni-project/virtual-host-gatherer
-Source0:        %{name}-%{version}.tar.gz
+Source0:        virtual-host-gatherer-git-0.0844d76.tar.gz
```

Comment:
I'm not so happy about having a hardcoded git commit SHA1 as the version number, doesn't it make more sense to tag a new release?

Reply:
I think this will be replaced by tito. I need to check if we can keep the old and tito will just replace it when we submit.

Reply:
It's automatically changed by tito when we run `rel-eng/build-packages-for-obs.sh`. I'll revert this to the previous `%{name}-%{version}.tar.gz`